resources/assets/uad_lists.json

```
@@ -1,3 +1,12 @@
+{
+  "com.dti.tracfone": {
+    "list": "Oem",
```

```suggestion
    "list": "Carrier",
```
If it's a carrier bloatware installer, shouldn't the list be `Carrier` then?